eks_fargate/README.md

```diff
@@ -109,10 +109,181 @@ You can run the Agent as a sidecar by using the [Datadog Admission Controller][3
 With manual configuration, you must modify every workload manifest when adding or changing the Agent sidecar. Datadog recommends you use the Admission Controller.
 
 <!-- xxx tabs xxx -->
-<!-- xxx tab "Admission Controller" xxx -->
-##### Admission Controller
+<!-- xxx tab "Datadog Operator" xxx -->
+##### Admission Controller using Datadog Operator
 
-<div class="alert alert-warning">This feature requires Cluster Agent v7.52.0+ and the <a href="http://docs.datadoghq.com/integrations/ecs_fargate">ECS Fargate integration</a>.
+<div class="alert alert-warning">This feature requires Cluster Agent v7.52.0+ and Datadog Operator v1.7.0+.
+</div>
+
+The setup below configures the Cluster Agent to communicate with the Agent sidecars, allowing access to features such as [events collection][29], [Kubernetes resources view][30], and [cluster checks][31].
+
+**Prerequisites**
+
+* Deploying RBAC is not required as the Operator creates the necessary RBACs and deploys the Cluster Agent.
```

Comment:
This is inaccurate, RBAC needs to be created in the applications namespaces as well since sidecar will run in those too (Operator doesn't have access or support creation RBAC there).

Reply:
@levan-m You are right, Yeah just roll back to the same comment with what it described in helm tab :)